azure/sdk-identity-resources-storage/AzureIdentityStorageExample.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>2eec9aab-2e9b-4211-82f7-98c5894764e9</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="dotnet-bot.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="dotnet-bot.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.2.0-preview.4" />
```

Comment:
I have concerns about publishing samples with preview bits, are there any stable versions of these at all?